docs/oracle/economic-guarantees.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Why the split into an additional economic guarantees page? Looks like the information is almost the same as the Economic Architecture page